tools/runners/test262-harness.py

```diff
@@ -405,6 +405,7 @@ def write(self, string):
         os.write(self.file_desc, string)
 
     def read(self):
+        # pylint: disable=undefined-variable
```

Comment:
Which variable is undefined here?

Reply:
pylint with python3 thinks that file is undefined variable. But test262-harness.py is still python2 script. (will be changed soon)

Reply:
Isn't this code just reading the file contents and returning? If yes the solution is simple: use "open" as a context manager read the data and return the contents.

Reply:
Yes. but it is completely unrelated to this PR.